cico_run_tests.sh

```diff
@@ -96,10 +96,7 @@ else
 fi
 
 # Build and push image
-# Use default length when not provided
-echo "${DEVSHIFT_TAG_LEN:=6}"
-VERSION_NUMBER=$(echo $GIT_COMMIT | cut -c1-${DEVSHIFT_TAG_LEN})
-TAG="SNAPSHOT-PR-${ghprbPullId}-${VERSION_NUMBER}"
+TAG="SNAPSHOT-PR-${ghprbPullId}"
```

Comment:
what is the limitation on cico for tagging?
and instead of removing the tag I would like to use 2 or 3 characters from the hash. we need tag each new build on new commit.

Reply:
Yes, agreed. But we cannot show custom tags on success comments. The CICO build uses the static success message.